modules/snooker/src/main/ts/ephox/snooker/api/CopyRows.ts

```diff
@@ -8,7 +8,7 @@ import * as Redraw from '../operate/Redraw';
 import { Generators } from './Generators';
 import { Warehouse } from './Warehouse';
 
-const copyRows = (table: SugarElement<HTMLTableElement>, target: TargetSelection, generators: Generators): Optional<SugarElement<HTMLTableRowElement>[]> => {
+const copyRows = (table: SugarElement<HTMLTableElement>, target: TargetSelection, generators: Generators): Optional<SugarElement<HTMLTableRowElement | HTMLTableColElement>[]> => {
```

Comment:
It's used in far too many places to be leaving comments unfortunately. I think this will just need to be one of those things people need to know since it's a confusing part of the HTML spec.